rest/admin_api.go

```diff
@@ -148,10 +148,27 @@ func (h *handler) handleReplicate() error {
 	}
 
 	params, cancel, _, err := h.readReplicationParametersFromJSON(body)
+	if err != nil {
+		return err
+	}
 
+	response, err := h.server.HTTPClient.Get(params.GetTargetDbUrl())
 	if err != nil {
 		return err
 	}
+	if response.StatusCode >= 400 {
+		return base.HTTPErrorf(response.StatusCode, "Unable to start replication target db not reachable")
+	}
+	response.Body.Close()
```

Comment:
response body needs closing on any nil error (even if the status code is >= 400)